GitUI/FontUtil.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace GitUI
+{
+    using System;
+    using System.Drawing;
+
+    public static class FontUtil
+    {
+        static FontUtil()
+        {
+            var hTheme = NativeMethods.OpenThemeData(IntPtr.Zero, "TEXTSTYLE");
+
+            var success = false;
+
+            if (hTheme != IntPtr.Zero)
+            {
+                // TODO: use C# 7.0 out var parameters
+                NativeMethods.LOGFONT pFont;
+                success = NativeMethods.GetThemeFont(hTheme, IntPtr.Zero, NativeMethods.TEXT_MAININSTRUCTION, 0, NativeMethods.TMT_FONT, out pFont) == NativeConstants.S_OK;
+
+                if (success)
+                {
+                    MainInstructionFont = Font.FromLogFont(pFont);
+
+                    NativeMethods.COLORREF pColor;
+                    success = NativeMethods.GetThemeColor(hTheme, NativeMethods.TEXT_MAININSTRUCTION, 0, NativeMethods.TMT_TEXTCOLOR, out pColor) == NativeConstants.S_OK;
+
+                    if (success)
+                    {
+                        MainInstructionColor = Color.FromArgb(pColor.R, pColor.G, pColor.B);
+                    }
+                }
+
+                NativeMethods.CloseThemeData(hTheme);
```

Comment:
Better to use `SafeHandle` https://msdn.microsoft.com/en-us/library/system.runtime.interopservices.safehandle(v=vs.110).aspx#Why

Reply:
Unfortunately there's no supplied [`SafeHandle` implementation](https://msdn.microsoft.com/en-us/library/microsoft.win32.safehandles(v=vs.110).aspx) for an `HTHEME` handle (i.e. one that calls `CloseThemeData`). I can implement one, but I think given the very short lifespan of this handle that we can probably get away without it. However, I've added a `finally` block to make sure the cleanup happens.